3_visualize/src/drought_learner_viz_components.R

```diff
@@ -1,31 +1,90 @@
 blank_plot <- function(streamflow_df, dv_tibble, growing_season = T){
   
+  axis_rects <- data.frame(
+    ymax = rep(-70, 6),
+    ymin = rep(-100, 6),
+    xmin = c(as.Date(sprintf("01/05/%s", focal_year),'%d/%m/%Y'),
+             as.Date(sprintf("01/06/%s", focal_year),'%d/%m/%Y'),
+             as.Date(sprintf("01/07/%s", focal_year),'%d/%m/%Y'),
+             as.Date(sprintf("01/08/%s", focal_year),'%d/%m/%Y'),
+             as.Date(sprintf("01/09/%s", focal_year),'%d/%m/%Y'),
+             as.Date(sprintf("01/10/%s", focal_year),'%d/%m/%Y')),
+    xmax = c(as.Date(sprintf("31/05/%s", focal_year),'%d/%m/%Y'),
+             as.Date(sprintf("30/06/%s", focal_year),'%d/%m/%Y'),
+             as.Date(sprintf("31/07/%s", focal_year),'%d/%m/%Y'),
+             as.Date(sprintf("31/08/%s", focal_year),'%d/%m/%Y'),
+             as.Date(sprintf("30/09/%s", focal_year),'%d/%m/%Y'),
+             as.Date(sprintf("14/10/%s", focal_year),'%d/%m/%Y'))
+  )
+  axis_rects_year <- data.frame(
```

Comment:
suggest creating this first and then filtering it to create the one written above so simplify.

Reply:
Good call!